.github/actions/test_deploy_macos.sh

```diff
@@ -0,0 +1,137 @@
+#!/bin/bash
+
+# Copyright (C) 2015, Wazuh Inc.
+# March 6, 2019.
+#
+# This program is free software; you can redistribute it
+# and/or modify it under the terms of the GNU General Public
+# License (version 2) as published by the FSF - Free Software
+# Foundation.
+
+# Global variables
+VERSION="$(sed 's/v//' src/VERSION)"
+MAJOR=$(echo "${VERSION}" | cut -dv -f2 | cut -d. -f1)
+MINOR=$(echo "${VERSION}" | cut -d. -f2)
+SHA="$(git rev-parse --short=7 "$1")"
+
+conf_path="/Library/Ossec/etc/ossec.conf"
+
+VARS=( "WAZUH_MANAGER" "WAZUH_MANAGER_PORT" "WAZUH_PROTOCOL" "WAZUH_REGISTRATION_SERVER" "WAZUH_REGISTRATION_PORT" "WAZUH_REGISTRATION_PASSWORD" "WAZUH_KEEP_ALIVE_INTERVAL" "WAZUH_TIME_RECONNECT" "WAZUH_REGISTRATION_CA" "WAZUH_REGISTRATION_CERTIFICATE" "WAZUH_REGISTRATION_KEY" "WAZUH_AGENT_NAME" "WAZUH_AGENT_GROUP" "ENROLLMENT_DELAY" )
+VALUES=( "1.1.1.1" "7777" "udp" "2.2.2.2" "8888" "password" "10" "10" "/Library/Ossec/etc/testsslmanager.cert" "/Library/Ossec/etc/testsslmanager.cert" "/Library/Ossec/etc/testsslmanager.key" "test-agent" "test-group" "10" )
+TAGS1=( "<address>" "<port>" "<protocol>" "<manager_address>" "<port>" "<password>" "<notify_time>" "<time-reconnect>" "<server_ca_path>" "<agent_certificate_path>" "<agent_key_path>" "<agent_name>" "<groups>" "<delay_after_enrollment>" )
+TAGS2=( "</address>" "</port>" "</protocol>" "</manager_address>" "</port>" "</password>" "</notify_time>" "</time-reconnect>" "</server_ca_path>" "</agent_certificate_path>" "</agent_key_path>" "</agent_name>" "</groups>" "</delay_after_enrollment>" )
+WAZUH_REGISTRATION_PASSWORD_PATH="/Library/Ossec/etc/authd.pass"
+
+function install_wazuh(){
+  echo "Testing the following variables $*"
```

Comment:
Leave a blank line before starting the code, please.

Reply:
Done [8f671ce](https://github.com/wazuh/wazuh/pull/15038/commits/8f671cecc18d6085330e70bc5a8d1957d49fecf8)